__tests__/app.test.js

```diff
@@ -65,3 +65,50 @@ describe("GET *", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Should respond with 200 status when successfully fetching an article", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+
```

Comment:
This might be a tiiiny bit superfluous because we are also doing this assertion in the test afterwards, but I'd rather see too many tests than too few :)